source/Calamari.Common/Plumbing/Extensions/VariableExtensions.cs

```diff
@@ -8,15 +8,12 @@ namespace Calamari.Common.Plumbing.Extensions
 {
     public static class VariableExtensions
     {
-        public static PathToPackage GetPathToPrimaryPackage(this IVariables variables, ICalamariFileSystem fileSystem, bool required)
+        public static PathToPackage GetPathToPrimaryPackage(this IVariables variables, ICalamariFileSystem fileSystem)
```

Comment:
I added the `required` flag on purpose, likely for when converting the other command to use this method